doc-content/enterprise-only/pages/building-custom-dashboard-widgets-creating-all-processes-heatmap-components-proc.adoc

```diff
@@ -0,0 +1,96 @@
+[id='building-custom-dashboard-widgets-creating-all-processes-heatmap-components-proc_{context}']
+
+= Creating processes heatmap components
+
+You can create heatmap components for multiple processes in {CENTRAL}.
+
+.Prerequisites
+
+* {KIE_SERVER} is deployed and connected to {CENTRAL}.
+* You have created a project in {CENTRAL} and it contains at least one business process asset.
```

Comment:
all processes heatmap you need multiple projects

Reply:
Done.